### PR TITLE
DKG Manifest Entry

### PR DESCRIPTION
**Pull request by ejfitzgerald**

### CODE REVIEW

File: libs/ledger/src/chain/block_coordinator.cpp [hunk 1]
```diff
@@ -507,9 +514,7 @@ BlockCoordinator::State BlockCoordinator::OnSynchronised(State current, State pr
   }
   else if (State::SYNCHRONISING == previous)
   {
-    FETCH_LOG_INFO(LOGGING_NAME, "Chain Sync complete on 0x", current_block_->body.hash.ToHex(),
-                   " (block: ", current_block_->body.block_number, " prev: 0x",
-                   current_block_->body.previous_hash.ToHex(), ")");
+    //
```
Comment (n-hutton) [inline libs/ledger/src/chain/block_coordinator.cpp:517]: delete

File: libs/muddle/src/muddle.cpp [hunk 2]
```diff
@@ -93,7 +93,7 @@ Muddle::Muddle(NetworkId network_id, CertificatePtr certificate, NetworkManager
 
 Muddle::~Muddle()
 {
-  MuddleRegistry::Instance().Unregister(node_address_);
+  MuddleRegistry::Instance().Unregister(this);
```
Comment (n-hutton) [inline libs/muddle/src/muddle.cpp:96]: I am curious about what this does

**Review by n-hutton — APPROVED** [review]
lgtm

File: libs/muddle/src/muddle.cpp (hunk 2 above)
Comment (ejfitzgerald) [reply libs/muddle/src/muddle.cpp:96]: Basically just removes an element from an internal map. The internal map contains weak pointers to muddle instances. The key for the map is the pointer to the instance (and that is all that it is used for).

**Pull request merged**